### PR TITLE
lbu: New module to manage Alpine Linux run-from-RAM configuration

### PR DESCRIPTION
**Pull request by kunkku**
Alpine Linux Local Backup Utility

##### SUMMARY
Module for Alpine Linux Local Backup Utility

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
lbu

### CODE REVIEW

File: lib/ansible/modules/system/lbu.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: lbu
+
+short_description: Local Backup Utility
```
Comment (Akasurde) [inline lib/ansible/modules/system/lbu.py]:
```suggestion
short_description: Local Backup Utility for Alpine Linux
```

File: lib/ansible/modules/system/lbu.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+version_added: "2.10"
+
+description:
+- Alpine Linux Local Backup Utility
```
Comment (Akasurde) [inline lib/ansible/modules/system/lbu.py]:
```suggestion
- Manage Alpine Linux Local Backup Utility while Alpine Linux is in a run-from-RAM configuration mode.
```

File: lib/ansible/modules/system/lbu.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+options:
+  commit:
+    description:
+    - Control whether to commit changed files
```
Comment (Akasurde) [inline lib/ansible/modules/system/lbu.py]:
```suggestion
    - Control whether to commit changed files.
```

File: lib/ansible/modules/system/lbu.py [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    type: bool
+  exclude:
+    description:
+    - List of paths to exclude
```
Comment (Akasurde) [inline lib/ansible/modules/system/lbu.py]:
```suggestion
    - List of paths to exclude.
```

File: lib/ansible/modules/system/lbu.py [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    type: list
+  include:
+    description:
+    - List of paths to include
```
Comment (Akasurde) [inline lib/ansible/modules/system/lbu.py]:
```suggestion
    - List of paths to include.
```

File: lib/ansible/modules/system/lbu.py [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+            'commit': {'type': 'bool'},
+            'exclude': {'type': 'list', 'elements': 'str'},
+            'include': {'type': 'list', 'elements': 'str'}
+        },
+        supports_check_mode=True
+    )
+
+    changed = False
+
+    def run_lbu(*args):
```
Comment (Akasurde) [inline lib/ansible/modules/system/lbu.py:89]:
```suggestion
    def run_lbu(*args):
        lbu_path = module.get_bin_path('lbu', required=True)
```

File: lib/ansible/modules/system/lbu.py [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+        code, stdout, stderr = module.run_command(['lbu'] + list(args))
```
Comment (Akasurde) [inline lib/ansible/modules/system/lbu.py]:
```suggestion
        code, stdout, stderr = module.run_command([lbu_path] + list(args))
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Thanks for the contribution.

**Review by Akasurde — APPROVED** [review]

**Comment by tdtrask** [comment]
shipit

**Pull request merged**